rest_framework/serializers.py

```diff
@@ -592,10 +592,6 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.child.bind(field_name='', parent=self)
 
-    def bind(self, field_name, parent):
-        super().bind(field_name, parent)
-        self.partial = self.parent.partial
-
```

Comment:
Aside: I was wondering if this should be left in, however, partial checking should be performed against the root serializer. So, this is safe to remove. 

Also, this probably would have been buggy in cases where the direct parent isn't also the root serializer...